Tests/FluentAssertions.Equivalency.Specs/SelectionRulesSpecs.cs

```diff
@@ -487,7 +487,7 @@ public void When_including_fields_it_should_succeed_if_just_the_included_field_m
             // Act
             Action act =
                 () =>
-                    class1.Should().BeEquivalentTo(class2, opts => opts.Including(_ => _.Field1).Including(_ => _.Field2));
+                    class1.Should().BeEquivalentTo(class2, opts => opts.Including(o => o.Field1).Including(o => o.Field2));
```

Comment:
🤔 I actually like to use `_` so we don't have to use an artificial name, but I can be convinced to change my mind. What does the rest think?

Reply:
I don't have a strong opinion on this. Maybe disable the rule?

Reply:
Wondering what @jnyrup thinks of this.

Reply:
I have to agree there, I find it strange a discarded parameter is used.

Reply:
So... I treat this as resolved, ok?